src/ensembl/src/content/app/custom-download/components/panel/Panel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { default as classNamesMerger } from 'classnames';
```

Comment:
Why this style? It was simply import classNames from 'classnames' before.

Reply:
That's done